lib/jsonapi_errors_handler/error_serializer.rb

```diff
@@ -8,15 +10,15 @@ def to_h
       serializable_hash
     end
 
-    def to_json(payload)
+    def to_json
```

Comment:
Have you checked that this won't change the behavior of how the Serializer is rendered?